patch/ros-jazzy-gz-cmake-vendor.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/gz-cmake-config.cmake.in b/gz-cmake-config.cmake.in
+index 56d06dd..c77c0b0 100644
+--- a/gz-cmake-config.cmake.in
++++ b/gz-cmake-config.cmake.in
+@@ -1,5 +1,7 @@
+ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+ 
++set(ENV{GZ_RELAX_VERSION_MATCH} "True")
```

Comment:
FYI, I think the `ENV{..}` CMake operator only works for reading env variables, see https://cmake.org/cmake/help/latest/variable/ENV.html .

Reply:
It's correct: https://cmake.org/cmake/help/latest/command/set.html#set-environment-variable

Reply:
Ah cool, I was confused by the docs in https://cmake.org/cmake/help/latest/variable/ENV.html that only mentions read.